Gemfile

```diff
@@ -60,7 +59,8 @@ gem 'jquery-rails', '~> 4.2.2'
 # To use debugger
 # gem 'ruby-debug19', :require => 'ruby-debug'
 
-gem 'json', '2.0.3' # For Ruby 2.3 compatibility
+#gem 'json', '2.0.3' # For Ruby 2.3 compatibility
+gem 'json'
```

Comment:
Should this be 'json', '~> 2.6'
2.6 is the latest, but we don't want to go to 3.0 if that gets published

Reply:
Indeed. Fixed.